src/tls13.c

```diff
@@ -1698,13 +1700,20 @@ static int ChaCha20Poly1305_Encrypt(WOLFSSL* ssl, byte* output,
 static int Tls13IntegrityOnly_Encrypt(WOLFSSL* ssl, byte* output,
                                       const byte* input, word16 sz,
                                       const byte* nonce,
+                                      const byte* iv, word16 ivSz,
                                       const byte* aad, word16 aadSz, byte* tag)
 {
     int ret;
 
-    /* HMAC: nonce | aad | input  */
+    (void)nonce;
+
+#ifdef WOLFSSL_TLS13_INTEGRITY_ONLY_NONCE
+    /* HMAC: [nonce] | iv | aad | input  */
```

Comment:
This still seems incorrect. The draft says:
```
HMAC(write_key, nonce || additional_data || plaintext)
```

Where nonce is not defined elsewhere, but I assumed it matches the per-record nonce in https://tools.ietf.org/html/rfc8446#section-5.3

Reply:
I've put the code back and not used the IV in the HMAC.
The HMAC key is write_key, see keys.c:2932-2955.